src/components/screens/IntegrationsCatalog/RecipesDetailScreen/RecipesDetailScreen.js

```diff
@@ -59,7 +60,18 @@ const ReadMe = styled.section`
   min-width: 0;
 `;
 
-const AddonsCallout = styled(Callout)`
+const AddonsCallout = styled.div`
+  padding: ${spacing.padding.medium}px;
+  border-radius: ${spacing.borderRadius.small}px;
+  display: flex;
+  flex-direction: column;
+
+  background: ${background.positive};
+  box-shadow: ${rgba(color.positive, 0.1)} 0 0 0 1px inset;
```

Comment:
Why apply custom styles instead of using `variant="positive"`?

Reply:
Because it's no longer the same component at this point that needs to be able to render addon lists.